railties/test/env_helpers.rb

```diff
@@ -21,6 +21,13 @@ def with_rack_env(env)
     end
   end
 
+  def with_port(port)
```

Comment:
can we wait with the helper extraction until it's needed in other cases as well. Once they're extracted it's harder to clean em up when tests get deleted or changed.